client/web/compose/src/components/ModuleFields/Editor/Geometry.vue

```diff
@@ -85,12 +85,23 @@
           <b-button
             variant="light"
             rounded
+            class="mr-2"
             @click="openMap"
           >
             <font-awesome-icon
               :icon="['fas', 'map-marked-alt']"
             />
           </b-button>
+
+          <b-button
+            variant="light"
+            rounded
+            @click="currentLocation"
```

Comment:
You also need to add this to the <multi> section above, since otherwise it wont work with multivalue geometry(you'll need to use index)